src/components/views/rooms/MessageComposerInput.js

```diff
@@ -252,33 +239,34 @@ module.exports = React.createClass({
                 self.onFinishedTyping();
             }
         }, 10); // XXX: what is this 10ms setTimeout doing?  Looks hacky :(
-    },
+    }
 
-    resizeInput: function() {
+    resizeInput() {
+        console.error('fixme');
```

Comment:
(before someone points this out) will remove most of what isn't needed once I'm done. `resizeInput()` isn't needed anymore since it's all handled in the styles.